src/pkg/packager/actions/actions.go

```diff
@@ -14,36 +15,43 @@ import (
 	"time"
 
 	"github.com/defenseunicorns/pkg/helpers/v2"
+	"github.com/goccy/go-yaml"
 	"github.com/zarf-dev/zarf/src/api/v1alpha1"
-	"github.com/zarf-dev/zarf/src/internal/packager/template"
+	ptmpl "github.com/zarf-dev/zarf/src/internal/packager/template"
```

Comment:
This is _not_ in scope for this PR but we do we have a few adjacent concepts of "templating" in Zarf's codebase. While we work on sunsetting variables I think it would be good to see if consolidating packager and variable templating into the `internal/template` package is viable.